src/components/ContactList/ContactList.js

```diff
@@ -1,20 +1,24 @@
 import React from 'react'
 import styled from 'styled-components/macro'
 import ContactListItem from '../ContactListItem/ContactListItem'
+import Header from '../Header/Header'
 
 export default function ContactList({ contacts }) {
   return (
+    <>
+    <Header />
```

Comment:
Is your header on every page? If so I would move the header to the app.js and place it outside the switch.